### PR TITLE
Add a gradle task to release a CrateDB Community Edition distribution. 

### PR DESCRIPTION
**Pull request by mkleen**
… where Enterprise related libs and plugins are excluded.

## Checklist

 - [ ] User relevant changes are recorded in ``CHANGES.txt``
 - [ ] Touched code is covered by tests
 - [ ] Documentation has been updated if necessary
 - [X] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

**Comment by mfussenegger** [comment]
Also looks like it is not creating a functional distribution:

```
↪  ls -1 ./crate-community-edition-4.0.0-SNAPSHOT-d5a9a2452a/
lib
logs
plugins
CHANGES.txt
LICENSE
NOTICE
README.rst
```

**Comment by mkleen** [comment]
> Maybe we could extend
> 
> [crate/devs/docs/basics.rst](https://github.com/crate/crate/blob/d5a9a2452a97e8c61004aa2e3b447946b1824c55/devs/docs/basics.rst#L65-L68)
> 
> Lines 65 to 68 in [d5a9a24](/crate/crate/commit/d5a9a2452a97e8c61004aa2e3b447946b1824c55)
> 
>  Build a tarball like so:: 
>   
>      $ ./gradlew distTar 
>   
> with two sentences to also mention `./gradlew communityEditionDistTar` ?

Done!

**Comment by mkleen** [comment]
> Also looks like it is not creating a functional distribution:
> 
> ```
> ↪  ls -1 ./crate-community-edition-4.0.0-SNAPSHOT-d5a9a2452a/
> lib
> logs
> plugins
> CHANGES.txt
> LICENSE
> NOTICE
> README.rst
> ```

Fixed!

**Comment by mkleen** [comment]
No ready yet.

File: app/build.gradle [hunk 1]
```diff
@@ -325,25 +327,65 @@ task chooseReleaseNotes(dependsOn: [':es:es-server:getVersion']) {
...
 }
-distZip.dependsOn(chooseReleaseNotes)
-
 
 installDist.dependsOn(chooseReleaseNotes)
 
-
 distributions {
+    communityEdition {
+        baseName = 'crate-community-edition'
```
Comment (mfussenegger) [inline app/build.gradle]: Maybe we could call this just `crate-ce` ? Not sure

Comment (seut) [reply app/build.gradle]: +1

File: devs/docs/basics.rst [hunk 2]
```diff
@@ -66,6 +66,10 @@ Build a tarball like so::
 
     $ ./gradlew distTar
 
+Build a tarball of the free Community Edition like so::
```
Comment (smakalias) [inline devs/docs/basics.rst]: I am wondering if we should remove the word `free` to align with what we market as options here:
https://crate.io/products/cratedb-editions

File: app/build.gradle (hunk 1 above)
Comment (mkleen) [reply app/build.gradle]: +1

**Pull request merged**